src/python/test/test_extract_expression.py

```diff
@@ -0,0 +1,558 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 DNAnexus, Inc.
+#
+# This file is part of dx-toolkit (DNAnexus platform client libraries).
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#   use this file except in compliance with the License. You may obtain a copy
+#   of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+
+# Run manually with python3 src/python/test/test_extract_expression.py
+
+import unittest
+import subprocess
+import sys
+import os
+import dxpy
+import json
+import shutil
+from dxpy_testutil import cd, chdir
+from dxpy.bindings.apollo.ValidateJSONbySchema import JSONValidator
+from dxpy.bindings.apollo.assay_filtering_json_schemas import (
+    EXTRACT_ASSAY_EXPRESSION_JSON_SCHEMA,
+)
+
+dirname = os.path.dirname(__file__)
+
+python_version = sys.version_info.major
+
+
+class TestDXExtractExpression(unittest.TestCase):
```

Comment:
Hi @JoeMulka, I would recommend creating more generic test helper methods so that individual cases won't need to get duplicated code for loading the file and asserting the result.